packages/patternfly-4/react-table/src/components/Table/Table.tsx

```diff
@@ -203,13 +203,17 @@ export interface IRow extends RowType {
 }
 
 export interface TableProps {
+  /** Adds accessible text to the Table */
```

Comment:
Might consider rewording this to say that it adds "_an accessible name for_" the Table